### PR TITLE
Remove SANSConv.ToWavelength and SANSScale

### LINKED ISSUE #26955: Remove SANSConvertToWavelength / Scale Workflow Algorithms

### PR DESCRIPTION
**Pull request by DavidFair**
**Description of work.**
Removes SANS ConvertToWavelength and SANSScale as part of the work fixing workspace history

Run the system and unit tests as following
- ctest -R SANS
- systemtest -R SANS

Process some SANS data:
- Download the ISIS training data from the project site
- Open the ISIS SANS GUI
- Open the LOQ folder and select the Mask file as the user file
- Open the batch file and select the CSV
- Hit process all

Fixes #26955   . 

*This does not require release notes* because this is part of an epic #26960 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by cailafinn — CHANGES_REQUESTED** [review]
System, unit, and functional tests pass. Just a few code and documentation comments.

File: dev-docs/source/ISISSANSReductionBackend.rst [hunk 1]
```diff
@@ -847,7 +846,8 @@ The dedicated work-flow algorithms for the SANS reduction are:
 - *SANSMove*
 - *SANSNormalizeToMonitor*
 - *SANSSave*
-- *SANSScale*
+- :ref:`Multiply <algm-Multiply>` (by Absolute Scale)
+- :ref:`Divide <algm-Divide>` (by Sample Volume)
```
[the remark this answers is not in the document]
Comment (DavidFair) [reply dev-docs/source/ISISSANSReductionBackend.rst]: No it's just coincidence as this is the order of algorithms executed. In later PRs we also break the fact it's alphabetic by chance

**Comment by DavidFair** [comment]
I'll put these back into alphabetical order with a fixup PR into this branch, as the various PRs will modify them out of order again (added into the epic #26960 )

**Review by cailafinn — APPROVED** [review]
Code looks good, unit and system tests still passing locally. Approved.

**Pull request merged**